src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -242,18 +236,18 @@ export class HaIntegrationCard extends LitElement {
                 "ui.panel.config.integrations.config_entry.system_options"
               )}
             </mwc-list-item>
-            ${!manifest
+            ${!this.manifest
               ? ""
               : html`<mwc-list-item>
                   <a
                     class="documentation"
-                    href=${manifest.documentation}
+                    href=${this.manifest.documentation}
                     rel="noreferrer"
                     target="_blank"
                     >${this.hass.localize(
                       "ui.panel.config.integrations.config_entry.documentation"
-                    )}</a
-                  >
+                    )}<ha-svg-icon path=${mdiOpenInNew}></ha-svg-icon
```

Comment:
Set it as a property, not an attribute:
```suggestion
                    )}<ha-svg-icon .path=${mdiOpenInNew}></ha-svg-icon
```

Reply:
Corrected!